stacks/TableRenamerStack.ts

```diff
@@ -0,0 +1,50 @@
+import { SubnetType } from "aws-cdk-lib/aws-ec2";
+import { Schedule } from "aws-cdk-lib/aws-events";
+import { PolicyStatement } from "aws-cdk-lib/aws-iam";
+import { Cron, Function, StackContext, use } from "sst/constructs";
+
+import { RdsStack } from "./RdsStack";
+import { VpcStack } from "./VpcStack";
+
+export function TableRenamerStack({ stack }: StackContext) {
```

Comment:
Rather than have this in its own stack, I would define it in the step function stack along with the other lambdas that are used in it.

It will then need adding onto the step function to run at the very end.